src/pages/home/HeaderView.tsx

```diff
@@ -151,6 +151,8 @@ function HeaderView({report, parentReportAction, onNavigationMenuButtonClicked,
     const policyDescription = getPolicyDescriptionText(policy);
     const isPersonalExpenseChat = isPolicyExpenseChat && isCurrentUserSubmitter(report);
     const hasTeam2025Pricing = useHasTeam2025Pricing();
+    // This is the part where we display the header text. Adding this condition is to prevent us from parsing HTML to text for user inputs
```

Comment:
```suggestion
    // This is used to ensure that we display the text exactly as the user entered it when displaying thread header text, instead of parsing their text to HTML. 
```